### PR TITLE
Add a test

### PR DESCRIPTION
**Pull request by kssenii**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit e61ec63e6fe7f6c028f9454f734168464183f282 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/55003/e61ec63e6fe7f6c028f9454f734168464183f282/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>✅ success</td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>⏳ pending</td></tr>
<tbody>
</table>

File: tests/queries/0_stateless/02344_describe_cache.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+# set -x
```
Comment (SmitaRKulkarni) [inline tests/queries/0_stateless/02344_describe_cache.sh:4]: maybe missed removing this ?

File: tests/queries/0_stateless/02344_describe_cache.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
```
Comment (SmitaRKulkarni) [inline tests/queries/0_stateless/02344_describe_cache.sh:2]: As this is cache test, should we add 'no-parallel' ?

Comment (kssenii) [reply tests/queries/0_stateless/02344_describe_cache.sh:2]: No, cache in this test is not shared between tests

**Review by SmitaRKulkarni — APPROVED** [review]
LGTM

**Pull request merged**